layouter/templates/layouter/bootstrap4/container.html

```diff
@@ -0,0 +1,28 @@
+{% load sekizai_tags %}
+{% load staticfiles %}
+<div class="row layouterRow--outer">
+    <div class="offset-lg-{{ instance.margin }} col-lg-{{ width }}">
+        <div class="row layouterRow--inner
+        {% if request.session.layouter_grid %}show-grid{% endif %}
+        {% if not instance.equal_height %}align-items-start{% endif %}
+        {{ instance.css_classes }}">
+            {% include 'layouter/bootstrap4/partials/one_col.html' %}
+            {% include 'layouter/bootstrap4/partials/two_cols.html' %}
+            {% include 'layouter/bootstrap4/partials/three_cols.html' %}
+            {% include 'layouter/bootstrap4/partials/four_cols.html' %}
+        </div>
+    </div>
+</div>
+{# The following JavaScript and Styles are only needed in editing mode. #}
```

Comment:
Since this part is actually duplicated in both `container.html` we should probably put this into an include.